discord/voice_client.py

```diff
@@ -623,13 +623,33 @@ def get_ssrc(self, user_id):
             user_id
         ]
 
+    @overload
     def play(
         self,
         source: AudioSource,
         *,
         after: Callable[[Exception | None], Any] = None,
-        wait_finish: bool = False,
+        wait_finish: Literal[False] = False,
     ) -> None:
+        ...
+
+    @overload
+    def play(
+        self,
+        source: AudioSource,
+        *,
+        after: Callable[[Exception | None], Any] = None,
+        wait_finish: Literal[True],
```

Comment:
```suggestion
        wait_finish: Literal[True] = True,
```

Reply:
This is not the case as `False` is implied but `True` should be passed explicitly.